internal/store/alert_configuration.go

```diff
@@ -35,10 +35,15 @@ type AlertConfigurationDeleter interface {
 	DeleteAlertConfiguration(string, string) error
 }
 
+type AlertConfigurationFieldsLister interface {
```

Comment:
[q] why not `MatcherFieldsLister` ? not a needed change but just wondering about this